spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/source/SparkScan.java

```diff
@@ -200,12 +220,31 @@ public CustomTaskMetric[] reportDriverMetrics() {
     }
 
     List<CustomTaskMetric> driverMetrics = Lists.newArrayList();
+    // common
     driverMetrics.add(TaskTotalFileSize.from(scanReport));
```

Comment:
Could you confirm `TaskTotalFileSize` represents the total size of read data?

If so, shall we call these metrics as `TaskTotalDataFileSize` and `TotalDataFileSize`? I know we follow the API from core but it seems a bit confusing. I had to look up the code to understand what this metric means. If we decide to rename, let's move it to the data files block below.